lib/DXIL/DxilModule.cpp

```diff
@@ -123,7 +123,7 @@ void DxilModule::SetShaderModel(const ShaderModel *pSM, bool bUseMinPrecision) {
   m_pSM->GetDxilVersion(m_DxilMajor, m_DxilMinor);
   m_pMDHelper->SetShaderModel(m_pSM);
   m_bUseMinPrecision = bUseMinPrecision;
-  m_pOP->SetMinPrecision(m_bUseMinPrecision);
+  m_pOP->InitWithMinPrecision(m_bUseMinPrecision);
   m_pTypeSystem->SetMinPrecision(m_bUseMinPrecision);
```

Comment:
You changed the `SetMinPrecision` call to `InitiWithMinPrecision` for the operations but not the type system. Should they not be consistent?

Reply:
Not necessarily.  I considered it, but looking into it, it doesn't really have the same justification.  There isn't initialization going on based on that in the type system, and it would have expanded the scope of changes, at which point, it looked like some other changes could be made to clean some code up that used these, but I didn't want more changes creeping into this PR.